### ISSUE TITLE
training and validation loss in brats 2022 challenge

### ISSUE THREAD

**Issue by abbas695** [NVIDIA/DeepLearningExamples#1307]
in your notebook you report that you trained 5 models for 130 epochs and averaged their predictions 
can i know what was the range of Val loss and train loss at the end of each fold?

**Comment by michal2409** [comment]
I'm sorry but I don't have this data for loss values. Please refer to this table to compare between BraTS2021 (Optimized U-Net) and BraTS 2022 (Tuned U-Net) for the mean dice score.

<img width="259" alt="Screenshot 2023-06-20 at 13 45 31" src="https://github.com/NVIDIA/DeepLearningExamples/assets/14821051/e31d5213-d4a4-4d60-be2f-bb865ce63ef7">

**Comment by abbas695** [comment]
thank you so much 
but does this table come from a certain paper?
i know the optimized U-net values come from the "Optimized U-Net for Brain Tumor Segmentation" paper 
but what is Tuned U-net?

**Comment by michal2409** [comment]
Tuned U-Net is the solution from BraTS 2022 notebook. Table is from our paper and will be available in proceedings (https://www.med.upenn.edu/cbica/brats)
